### ISSUE TITLE
Conditional Decayed="False" does not work.

### ISSUE THREAD

**Issue by phillipinman** [Regalis11/Barotrauma#12362]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

Hi,

This is a continuation of a previous issue that I believe was fixed in the latest patch but my local testing implies that it may not be.

When I try to make a custom seed that uses a status effect to supply power to a pump component the Conditionals do not appear to work or are ignored.

XML snippet:

```
    <Growable>  
      # Omitted Plant stuff.
      <StatusEffect type="OnContained" target="This" Voltage="1.0" setvalue="true" Comparison="And">
        <Conditional Decayed="false"/>
        <Conditional FullyGrown="true"/>
      </StatusEffect>
    </Growable>
    <Pump FlowPercentage="-100" canbeselected="false" maxflow="100" MinVoltage="1" IsOn="true"/>
```

I have tested with even just "Decayed" without the "FullyGown" conditional and the issue persists. Even using just "FullyGrown" results in the pump working in the players inventory without being planted.

What I expect to happen is as long as the plant is fully grown and not decayed (dead) the statuseffect should evaluate to true and supply power to the pump.

It is entirely possible this is an issue with my logic and if so please feel free to mark this as closed, any additional insight as to why this is failing would also be welcome :)

If you can also just confirm that "Decayed" and "FullyGrown" is a usable conditional that would help heaps :)

Link to previous issue: https://github.com/Regalis11/Barotrauma/issues/11995

Test Setup, in this image the water is being drained regardless of the plants condition.
![image](https://github.com/Regalis11/Barotrauma/assets/11908363/f6edd3be-fdfc-415f-808f-f61b0ed1cc86)

Note: No mods besides my own are in use.

### Reproduction steps

1. Create a seed item using the above XML snippet.
2. Plant the seed in game in a planter.
3. Flood the room using "editwater"
4. Wait for the seed to "Decay/Wither"
5. Observe that the conditional "Decayed="False" does not in fact trigger and it keeps supplying voltage to the pump.

### Bug prevalence

Happens every time I play

### Version

v1.0.21.0

### Which operating system did you encounter this bug on?

Windows

### Relevant error messages and crash reports

```shell
NA
```

**Comment by Regalis11** [comment]
Thank you for the report!

Looks like I should've tested the example XML I posted in the previous ticket - FullyGrown and Decayed hadn't actually been exposed to StatusEffects so the effect did not work. I exposed them now in https://github.com/Regalis11/Barotrauma-development/commit/b29cd17bb95aec813b6d91c41fba55ff4812d14f (commit in our private repo).

I also noticed another issue while testing this: the pump doesn't have any power consumption defined, which means it will default to 0 and the pump will work regardless of the voltage. Adding something like `PowerConsumption="1"` to the XML should fix that.
